internal/cmd/build-iso.go

```diff
@@ -0,0 +1,162 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/kairos-io/AuroraBoot/deployer"
+	"github.com/kairos-io/AuroraBoot/pkg/schema"
+	"github.com/spectrocloud-labs/herd"
+	"github.com/urfave/cli/v2"
+)
+
+const (
+	KairosDefaultArtifactName = "kairos"
+)
+
+var BuildISOCmd = cli.Command{
+	Name:    "build-iso",
+	Aliases: []string{"b"},
+	Usage:   "Builds an ISO from a container image or github release",
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:    "cloud-config",
+			Aliases: []string{"c"},
+			Usage:   "The cloud config to embed in the ISO",
+		},
+		&cli.StringFlag{
+			Name:    "name",
+			Aliases: []string{"n"},
```

Comment:
put the KairosDefaultArtifactName here as a default value if nothing is passed?

Reply:
good point. Done.